controllers/authentication.js

```diff
@@ -4,12 +4,21 @@
 const path = require('path')
 
 const auth = {
+
+   // nodeboard/api/login/
```

Comment:
should login and logout be in api? given we are using facebook this means we make a api call just to be redirected to facebook. Maybe this would be better in it's own server side route?

Reply:
login is misnamed. It should really be "login/success. Logout could be under auth/logout.

As for the final one that's just serving the main page which is the default route.

Reply:
This is also more of a open ended question, i think we can easily justify this being in the api but since we have to have a index route anyway maybe just having that serve the home rather then going through the api makes sense?